src/GrpcProtos/capture.proto

```diff
@@ -325,6 +325,24 @@ message SystemMemoryUsage {
   int64 cached_kb = 6;
 }
 
+message ProcessMemoryUsage {
+  uint32 pid = 1;
+  uint64 timestamp_ns = 2;
+  // These fields are retrieved from /proc/<pid>/status.
+  int64 rss_kb = 3;
```

Comment:
I would use the exact same names as in /proc/pid/status, adjusted to our code style. Adding the unit at the end is a good idea, though. That is, this should be `vm_rss_kb`.

Reply:
Make sense! Changed to use `vm_rss_kb`. Thanks for the suggestion!